### DISCUSSION TITLE
Search results are blocked by the CORS policy

### DISCUSSION THREAD

**Discussion by ereator** [osm-search/Nominatim#2488] [Q&A]
Dear all,
Recently I've got one problem with fetching the data from nominatim.openstreetmap.org:
> Access to fetch at 'https://nominatim.openstreetmap.org/search?accept-language=de&countrycodes=de&addressdetails=0&limit=5&email=sergej.kosov%40gmail.com&q=Hannover&format=json' from origin 'https://www.uvvo.club' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

May I ask you to add `Access-Control-Allow-Origin: *` to your server or provide any other solution to fix the problem?

Thank you in advance...

**Comment by lonvia** [comment]
When you see a CORS issue like that with nominatim.openstreetmap.org then the underlying issue is likely that you have been blocked for some reason. The site sends proper CORS headers with normal responses but it does not send them when returning a HTTP 403 or 429. Check your logs for HTTP errors.

Also note #2485.

**Reply by ereator** [reply]
Thank you for your response. 
The error appears not every time, but in approximately 25 - 33 % of requests. In that cases I  get HTTP 502:
![image](https://user-images.githubusercontent.com/18093984/138771011-5f6917da-0c68-4299-8ace-0ff8addd7349.png)

**Reply by mtmail** [reply]
nominatim.openstreetmap.org is currently running at reduced capacity. It can't handle all requests and that leads to HTTP 502 responses. See https://github.com/osm-search/Nominatim/issues/2485#issuecomment-951151227

**Reply by ereator** [reply]
Thank you! That explains my issue 👍

**Comment by SanchoPanzaaa** [comment]
I would like to ask, if there are any other known issues with CORS ? 

As firstly our connection works, 
after few days CORS problem appears and lasts.

![image](https://user-images.githubusercontent.com/97128098/220180297-f47b6e1f-9f9c-48cd-ba7e-05b51d6d58a7.png)

It's constant state, and we are unable know the reason.

Might you assist or give some explanation of why this may occurred ?

**Reply by mtmail** [reply]
You posted this as an answer to a 2 year old question. If it's a question open a new discussion.
